src/main/java/unicash/model/transaction/DateTime.java

```diff
@@ -66,13 +80,13 @@ public LocalDateTime getDateTime() {
     }
 
     /**
-     * Helper method to stringify LocalDateTime objects into original text string
-     * pass by the user.
+     * Helper method to return original text string
+     * passed by the user.
      *
      * @return text string of the LocalDateTime object
      */
     public String originalString() {
-        return dateTime.format(DateTimeFormatter.ofPattern(DATETIME_PATTERN));
+        return originalDateTime;
```

Comment:
Inline with the comment in #96 where `originalString` might seen misleading, maybe we can rename this to `inputString` or something if we're looking for uniformity in naming